src/redux/sagas/packing.ts

```diff
@@ -140,6 +141,7 @@ function* submitShipmentItems(action: any) {
     yield hideScreenLoading();
   } catch (e) {
     console.log('function* submitShipmentItems', e.response);
+    showPopup({message: "Fail to submit ShipmentItems "+e.response, positiveButton: "ok"});
```

Comment:
Remove console.log in the line above.
Use `Fail to submit ShipmentItems ${e.response}` as a message.
Replace double-quotes with single-quotes.